docs/backend-api/backend-api-installation.md

```diff
@@ -0,0 +1,30 @@
+# Backend API Installation
+
+Require `shopsys/backend-api` package in the `composer.json` or run
+
+```bash
+composer require shopsys/backend-api ^8.0
+```
+
+Backend API integration is fully automated, so you have to run only (you have to be in the project root directory)
+
+```bash
+bash vendor/shopsys/backend-api/integration/integrate.sh
```

Comment:
what about disintegrate?

Reply:
I assume programmers will do only the integration. If they need revert their work, they have `git checkout -- ...`